docs/trex_release-notes.md

```diff
@@ -11,9 +11,44 @@ layout: docs
 ----
 See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
+----
+
+### Tableau Dashboard Extensions API version 1.5
+
+*June 2021*
+
+* Tableau Dashboard Extensions API library: `tableau.extensions.1.5.0.js` <br>(download or clone the Extensions API repository on [GitHub](https://github.com/tableau/extensions-api){:target="_blank"}.) <br/>
+
+About this release:
+
+* The `Filter.getFieldAsync` method now works as expected and properly returns the field. This fix requires Tableau 2019.2 and later.
+
+* Show/Hide (`setZoneVisibilityAsync`) can now be applied to any dashboard zone.
+
+* The `selectMarksByValueAsync` method now supports combined selection criteria types (bug fixed).
+
+* The following are all improvements to the `getSummaryData` method:
```

Comment:
For user documentation, we should always spell out getSummaryDataAsync, so they know the exact method. (Change all from getSummaryData to getSummaryDataAsync)

Reply:
Yikes! Thanks for catching that, John. I'll fix all the instances and add a link to the reference page.